app/models/digital_object/publish_target.rb

```diff
@@ -145,7 +145,11 @@ def publish_digital_object(digital_object, do_ezid_update)
       Authorization: "Token token=#{publish_target_field('api_key')}"
     )
     if do_ezid_update && response.code == 200 && response.headers[:location].present?
-      digital_object.do_ezid_publish(response.headers[:location])
+      # By this point, all records should have an ezid. Let's update the status of
+      # that ezid to :public, and send the latest published_object_url in case it changed.
+      unless update_doi_metadata(response.headers[:location])
```

Comment:
@elohanlon Should this line have been `digital_object.do_ezid_publish(response.headers[:location])`

Reply:
@cgalarza You're right.  Fixing that now and will commit/push change very soon.